stdlib/importlib/machinery.pyi

```diff
@@ -1,6 +1,7 @@
 import importlib.abc
 import sys
 import types
+from builtins import type as Type  # alias to avoid name clashes with fields named "type"
```

Comment:
I'm so confused. Why do we need the alias? There don't seem to be any fields named `type` in this module — why are potential name clashes an issue?

Reply:
Cause all the other parts in the code that uses `builtins.type` does it that way. Whenever I'm in doubt I grep the source to find out what the established convention is.